themes/default/content/blog/next-level-iac-package-ecosystems/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Next-level IaC: Powering Up Pulumi with Package Ecosystems"
+allow_long_title: true
+date: 2024-05-08
```

Comment:
hey @thoward ; you'll want to modify the date for Monday our new target publish date